.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: default
-          toolchain: '1.81'
+          toolchain: '1.82'
```

Comment:
DataFusion 46 requires Rust 1.82

You can see CI fail without these changes
https://github.com/delta-io/delta-rs/actions/runs/13591787541/job/38000195037?pr=3261

I don't know what the MSRV policy in delta is so we probably can't merge this PR until it is ok to increase MSRV in delta